### PR TITLE
GT 310

### PR DESCRIPTION
**Pull request by dsgoers**
This table is redundant because we generate what we need dynamically

### CODE REVIEW

File: db/migrate/20170530201527_remove_one_sky_phrases_from_database.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class RemoveOneSkyPhrasesFromDatabase < ActiveRecord::Migration[5.0]
+  def change
+    drop_table :onesky_phrases do |t|
+      t.string :text, null: false
+      t.string :onesky_id, index: { unique: true }, null: false
+      t.references :page, null: false
```
Comment (knutsenm) [inline db/migrate/20170530201527_remove_one_sky_phrases_from_database.rb]: Does the above make this reversible? Cool.

File: app/models/onesky_phrase.rb [hunk 2]
```diff
@@ -1,20 +1,10 @@
 # frozen_string_literal: true
 
-class OneskyPhrase < ActiveRecord::Base
-  belongs_to :page
+class OneskyPhrase
```
Comment (knutsenm) [inline app/models/onesky_phrase.rb]: Why is this class needed at all?

File: db/migrate/20170530201527_remove_one_sky_phrases_from_database.rb (hunk 1 above)
Comment (dsgoers) [inline db/migrate/20170530201527_remove_one_sky_phrases_from_database.rb]: Yup 😄

**Comment by dsgoers** [comment]
thanks!!

**Pull request merged**